### ISSUE TITLE
Some filesystems are case sensitive

### ISSUE THREAD

**Issue by jamesjer** [gap-packages/polycyclic#24]
Trying to build the documentation for version 2.12 on a Linux system:
```
Generating documentation in Directory("./doc/")
#I Composing XML document . . .
Collect( res, src, NormalizedFilename( main ), 0 
 ); at /usr/lib/gap/pkg/GAPDoc/lib/ComposeXML.gi:247 called from
ComposedDocument( "GAPDoc", path, Concatenation( main, ".xml" ), files, true 
 ) at /usr/lib/gap/pkg/GAPDoc/lib/Make.g:43 called from
CallFuncList( makeDocFun, args 
 ); at /usr/lib/gap/pkg/AutoDoc/gap/Magic.gi:573 called from
<function "AutoDoc">( <arguments> )
 called from read-eval loop at line 10 of makedoc.g
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
Error, Cannot include file ./doc/Polycyclic.xml.
```
The name of the file is actually doc/polycyclic.xml, with a small 'p'.

**Comment by jamesjer** [comment]
I should have mentioned that this is the tail of the output from running "gap makedoc.g" with gap 4.8.10.

**Comment by fingolfin** [comment]
Thanks for the report. Fixed in f1e78d19ecc4d4801b711af0980fde4f27421cfd and the fix will be in the next release.
